build.sbt

```diff
@@ -339,9 +352,19 @@ val mtagsSettings = List(
     val scalaVersionsWithSpecialCompat = Set("2.13.9", "2.13.10", "2.13.11")
     if (scalaVersionsWithSpecialCompat(scalaVersion.value))
       current.filter(f => f.getName() != "scala-2.13")
+    else if (isScala3PresentationCompiler(scalaVersion.value))
+      List(base / s"scala-3-presentation-compiler")
     else
       current
   },
+  Test / unmanagedSourceDirectories ++= {
```

Comment:
What classes do we need this for?

Reply:
We need this for testing. Cross tests rely on mtags, docstring, global indexes and jdksources.

Reply:
Could we not just add this directory to the test scope in the tests? This looks hard to follow since you add the directory here, but then it's actually only needed for tests.